src/components/ProductList.jsx

```
@@ -19,14 +19,54 @@ function ProductList() {
+        method: "DELETE",
+      });
+      fetchProducts();
+    } catch (error) {
+      console.error("fetch 실패: ", error);
+    }
   };
 
   const handleAdd = async (newProduct) => {
     // TODO: POST API를 호출하고 fetchProducts() 호출
+    try {
+      const res = await fetch(API_URL, {
```

ESLint

[error] 'res' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)